merlin/dag/ops/udf.py

```diff
@@ -78,7 +79,8 @@ def transform(
             else:
                 # shouldn't ever happen,
                 raise RuntimeError(f"unhandled UDF param count {self._param_count}")
-        return new_df
+        # return input type data
+        return make_df(new_df)
```

Comment:
This works ok for data frame based  graphs, but we might need to revisit in a follow-up change to make this support tensortable too?

Reply:
No, I think its fine, for tensortable it also works because the executor handles the change to something compatible for the next operator. https://github.com/NVIDIA-Merlin/core/blob/main/merlin/dag/executors.py#L105

Reply:
what about if a user-defined function returns a TensorColumn type, is that something we'd like to support?